predicators/envs/base_env.py

```diff
@@ -308,6 +320,35 @@ def _parse_language_goal_from_json(
         goal_spec = json.loads(response)
         return self._parse_goal_from_json(goal_spec, id_to_obj)
 
+    def _parse_goal_from_input_to_json(
+            self, init_state: State, json_dict: Any,
+            object_name_to_object: Dict[str, Object])\
+            -> Set[GroundAtom]:  # pragma: no cover
+        """Helper for parsing language-based goals from terminal input."""
+        json_dict["init"] = init_state
+        for obj, _ in init_state.data.items():
+            object_name_to_object[obj.name] = obj
+        print("\n\nInit State:", init_state.simulator_state, "\n")
```

Comment:
This is potentially really unsafe! It will only work for a `_PDDLEnv` that state will have a `.simulator_state`!